bindings/libdnf5/advisory.i

```diff
@@ -39,6 +39,7 @@
 
 #define CV __perl_CV
 
+%include "libdnf/base/base_weak.hpp"
```

Comment:
Maybe it could be added into the `common.i` file and not including it in each `*.i`?

Reply:
That sounds good, I have updated the PR.
Thanks.